compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/generators/ClassMemberGenerator.kt

```diff
@@ -64,6 +64,14 @@ internal class ClassMemberGenerator(
                     else -> null
                 }
             }
+            irClass.declarations.filter {
```

Comment:
Maybe add some comments? I assume:
```
// Add delegated members *before* fake override generations.
// Otherwise, fake overrides for delegated members, which are redundant, will be added.
```
or something like that.

Reply:
done